core/src/main/scala/com/eharmony/spotz/optimizer/random/RandomSearchResult.scala

```diff
@@ -17,15 +17,7 @@ class RandomSearchResult[P, L](
   extends OptimizerResult[P, L](bestPoint, bestLoss) {
 
   override def toString = {
```

Comment:
Same reason, I'm overriding the case class' toString.  I don't have a need for the default one and prefer using the toString method for my own impl